ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -576,7 +576,7 @@ export default defineComponent({
         scrollToInvalid()
         return
       }
-      if ((isRoleStaffReg) || (isRoleStaffSbc)) {
+      if ((isRoleStaffReg.value) || (isRoleStaffSbc.value)) {
```

Comment:
How was it working before without getting the values?

Reply:
More or less in JS would resolve as check 'Does this property exist?' yes, return true.  vs checking the refs value, which is actually a property that exists on the ref. 

ie isRoleStaffReg === { value: someBoolean }.